group_vars/matrix_servers

```diff
@@ -1791,6 +1791,8 @@ matrix_ssl_domains_to_obtain_certificates_for: |
     +
     ([matrix_server_fqn_ntfy] if matrix_ntfy_enabled else [])
     +
+    ([matrix_bot_postmoogle_domain] if matrix_bot_postmoogle_enabled else [])
```

Comment:
```suggestion
    ([matrix_bot_postmoogle_domain] if matrix_bot_postmoogle_enabled and matrix_bot_postmoogle_domain != matrix_server_fqn_matrix else [])
```

---

It might be a good idea to add ` | uniq` (or is it ` | unique` or something else?) somewhere as well. It could be in the actual `with_items` iteration in `roles/matrix-nginx-proxy`. This way, even if someone unintentionally duplicates domains in `matrix_ssl_domains_to_obtain_certificates_for` (or `matrix_ssl_additional_domains_to_obtain_certificates_for` used within it), we won't be doing double work.

Reply:
Good point about `unique` filter, but I doubt the `and` condition needed in the group_vars - it's already hard to read, so let's just leave one `if` condition and apply `unique` filter when needed

Reply:
Each thing is on a line of its own, so it's not like it's becoming very complicated. Given that we have ` | unique` now, I suppose we're good and can keep this simple.